spec/network_x/traversals/dfs_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
```

Comment:
These `spec_helper` requires are absolutely not required. Try running the specs without them. 😄

Reply:
Okay, will remove it. Also, is it necessary to write specs for each and every function? I haven't written specs for some straightforward functions, like the ones which calculate path lengths. If the paths calculated by them are correct, the path lengths would be correct too.